packages/server/server/api/destinations/destination.interface.ts

```diff
@@ -0,0 +1,24 @@
+export enum DestinationCategory {
+    FOOD = 'food',
+    ACCOMMODATION = 'accommodation',
+    ENTERTAINMENT = 'entertainment',
+    SHOPPING = 'shopping',
+    SIGHTSEEING = 'sightseeing',
+}
+
+export default interface Destination {
+    uuid: string;
+    name: string;
+    category: DestinationCategory;
+    geometry: {
+        lat: number;
+        lon: number;
+    };
+    formattedAddress: string;
+    formattedPhoneNumber: string;
+    website: string;
```

Comment:
ohh I forgot to mention one thing, google place detail has no description. So I used the website to replace the description on destination card lol. Dun know what else we could have use there

Reply:
Yea, realized that too. Seems like there's an issue request Google to add description back to their api, but pretty sure it's not gonna be addressed anytime soon. I think I will leave the description field in the model, but we won't display it anywhere I guess LOL.